src/main/java/pro/sky/telegrambotanimalshelter/service/implementation/CatServiceImpl.java

```diff
@@ -0,0 +1,68 @@
+package pro.sky.telegrambotanimalshelter.service.implementation;
+
+
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import pro.sky.telegrambotanimalshelter.exceptions.CatNotFoundException;
+import pro.sky.telegrambotanimalshelter.models.Cat;
+import pro.sky.telegrambotanimalshelter.repository.CatRepository;
+import pro.sky.telegrambotanimalshelter.service.interfaces.CatService;
+
+import java.util.Collection;
+
+
+@Service
+public class CatServiceImpl implements CatService {
+
+    private final CatRepository repository;
+
+    private static final Logger logger = LoggerFactory.getLogger(CatServiceImpl.class);
```

Comment:
```suggestion
    private static final Logger LOGGER = LoggerFactory.getLogger(CatServiceImpl.class);
```
поля static И final надо писать в ВЕРХНЕМ регистре